data/language/en-GB.txt

```diff
@@ -3842,3 +3842,6 @@ STR_7000    :or
 STR_7001    :Ride name
 STR_7002    :{STRINGID} {STRINGID}
 STR_7003    :Audio file ‘{STRING}’ is truncated. Expected sample {INT32}, but only {INT32} are available. Consider reinstalling RCT2.
+STR_7004    :Could not start replay, file ‘{STRING}’ not found
```

Comment:
7004 and 7005 are pretty much identical, I think these should be merged, but note one of the comments down below